Lib/fontTools/ttLib/ttFont.py

```diff
@@ -487,6 +487,14 @@ def getGlyphOrder(self):
 				# in combination with the Adobe Glyph List (AGL).
 				#
 				self._getGlyphNamesFromCmap()
+			elif len(glyphOrder) < self['maxp'].numGlyphs:
+				#
+				# Not enough names found in the 'post' table.
+				# Can happen when 'post' format 1 is improperly used on a font that
+				# has more than 258 glyphs (the lenght of 'standardGlyphOrder').
+				#
+				log.warning("'post' table format 1 discarded; font has more than 258 glyphs")
```

Comment:
the warning mentions format 1 or font having more than 258 glyphs, but I don't see that you actually check any of these two specific conditions. You simply test if the glyphOrder that returned from post table is smaller than the total number of glyphs in the font according to maxp. Could it also be an invalid truncated post format 2 that contains less glyphs than there are in maxp.numGlyphs? (i'm speculating, I actually don't know if that's even possible). Maybe better to simply warn "Not enough names found in the 'post' table, generating them from cmap instead".